app/views/notes/_note.gpx.builder

```diff
@@ -1,9 +1,14 @@
 xml.wpt("lon" => note.lon, "lat" => note.lat) do
+  xml.time note.created_at.to_s(:iso8601)
+  xml.name "#{t'note.mine.description'} #{note.id}"
```

Comment:
You're right.
Sorry, I did not look properly for the suitable translation. I searched for "Hinweis" in the German translations, which is the proper translation for "note", but unfortunately the translation does not go the direct way in the user notes page ("description" normally is just "Beschreibung"), which led to this bad choice.

I'll update the PR.